control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -2497,9 +2499,22 @@ components:
           type: object
           additionalProperties:
             type: string
+        affinity  :
+          example: ''
+          description: |-
+            Includes resources with the same $label eg:
```

Comment:
```suggestion
            Includes resources with the same $label key eg:
```

Reply:
Does this need to be a map? Can't it just be a set or a list?

Reply:
I think a map is better then we would not need to parse to map again at some point?

Reply:
hmm why do we need it as a map?
Once the key is set, don't we simply set equivalent key/val on the include map?

Reply:
Map makes it look consistent , yes once the key is set we will set the equivalent key/value on include map

Reply:
It may be consistent, but include and exclude have to work with a map as they might have values. Here affinity will never have a value? So Set would be preferable IMHO as it clearly states the behaviour. 
But if you think map is better then at the very least lets ensure that value is always empty string when we process the request from rest.